### PR TITLE
update zh-TW translation

### PR DESCRIPTION
**Pull request by 5idereal**

### CODE REVIEW

**Comment by simi** [comment]
Hello and thanks for the pull request.

@adrianthedev Would it make sense to contribute avo translation into https://github.com/avo-hq/avo/tree/main/lib/generators/avo/templates/locales?

@khiav223577 @JuanitoFatas @hrs113355 any chance to review?

File: config/locales/zh-TW.yml [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
+  footer_about_html:
+    RubyGems.org 是 Ruby 社群的 Gem 套件管理服務，讓您能立即地<a href="%{publish_docs}">發佈</a>及<a href="%{install_docs}">安裝</a>您的 Gem 套件，並且利用
+    <a href="%{api_docs}">API</a> 查詢及操作<a href="%{gem_list}">可用 Gem</a> 的詳細資訊。<br />現在就<a href="%{contributing_docs}">成為貢獻者</a>，貢獻一己之力來改善本站。
   footer_sponsors_html:
+    RubyGems.org 透過與廣大的 Ruby 社群合作而誕生。
+    <a href="https://www.fastly.com/">Fastly</a> 提供頻寬和 CDN 支援，
+    <a href="https://www.rubycentral.org/">Ruby Central</a> 支付設備費用，並為進行中的開發和營運工作提供資金。
+    <a href="%{sponsors_page}">進一步了解我們的贊助商和它們是如何合作的</a>。
   footer_join_rt_html:
+    我們需要您的幫助來資助開發者為保證 RubyGems.org 能夠順利運行所耗費的時間。
```
Comment (khiav223577) [inline config/locales/zh-TW.yml:17]: MINOR: 覺得這一整句有點太長了
這樣看能不能更容易理解一點：

```suggestion
    我們需要您的資助來幫助開發者有更多開發時間，讓 RubyGems.org 能夠持續穩定地運行下去。
```

**Comment by khiav223577** [comment]
Others LGTM

**Comment by simi** [comment]
@5idereal can you please check on suggested changes?

**Comment by 5idereal** [comment]
Hi @simi, I've committed all the suggestions from @khiav223577.

@khiav223577 感謝您的 review!

**Comment by simi** [comment]
Thanks! There were also some troubles on CI due to formatting, let's wait for other CI round.

File: config/locales/zh-TW.yml [hunk 2]
```diff
@@ -178,121 +185,124 @@ zh-TW:
...
       title:
     ownership_confirmation:
-      subject:
-      title:
-      subtitle:
-      body_text:
+      subject: 請確認 RubyGems.org 上 %{gem} Gem 的所有權
+      title: 所有權確認
+      subtitle: 嗨 %{handle}！
+      body_text: %{authorizer} 把您加入了 %{gem} Gem 的擁有者名單。請點擊下方連結來確認您的所有權。
```
Comment (khiav223577) [inline config/locales/zh-TW.yml:262]: 這行似乎有問題，包個雙引號看看？
```suggestion
      body_text: "%{authorizer} 把您加入了 %{gem} Gem 的擁有者名單。請點擊下方連結來確認您的所有權。"
```

error message:
```
I18n::InvalidLocaleData: can not load translations from /home/runner/work/rubygems.org/rubygems.org/config/locales/zh-TW.yml: #<Psych::SyntaxError: (<unknown>): found character that cannot start any token while scanning for the next token at line 262 column 18>
```

**Comment by adrianthedev** [comment]
Yes @simi. That would be great to have it in Avo. 
Thank you!

**Comment by simi** [comment]
@5idereal sorry for the delay on this PR, can you please rebase? I'm happy to merge once done.

**Comment by 5idereal** [comment]
@simi should be ready for merge. can you check it again please?

**Comment by simi** [comment]
@5idereal sorry for long delay :( Any chance to rebase this for final time please?

**Comment by simi** [comment]
:information_source: I tried to rebase on my own :pray:

**Comment by simi** [comment]
Thanks anyone involved in here and sorry for the delay. I have fixed small problems with keys, rebased and finally merged.

**Pull request merged**